0.17.x Not working on Windows (or at least, not on Appveyor)

https://ci.appveyor.com/project/MoOx/phenomic/build/2886#L2514

```
 ModuleBuildError: Module build failed: Error: Cannot find module 'C:projectsphenomic
egister.js'
    at Function.Module._resolveFilename (module.js:455:15)
    at Function.Module._load (module.js:403:25)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\projects\phenomic\test-phenomic-theme-base\node_modules\css-loader\index.js!C:\projects\phenomic\test-phenomic-theme-base\node_modules\postcss-loader\index.js!C:\projects\phenomic\test-phenomic-theme-base\src\index.global.css:1:1)
    at Module._compile (module.js:556:32)
    at Object.exec (C:\projects\phenomic\node_modules\webpack-core\lib\NormalModuleMixin.js:88:7)
    at Object.<anonymous> (C:\projects\phenomic\test-phenomic-theme-base\node_modules\extract-text-webpack-plugin\loader.js:97:22)
    at Compiler.<anonymous> (C:\projects\phenomic\node_modules\webpack\lib\Compiler.js:214:10)
    at C:\projects\phenomic\node_modules\webpack\lib\Compiler.js:403:12
    at Compiler.next (C:\projects\phenomic\node_modules\tapable\lib\Tapable.js:67:11)
    at Compiler.<anonymous> (C:\projects\phenomic\test-phenomic-theme-base\node_modules\extract-text-webpack-plugin\loader.js:78:5)
    at Compiler.next (C:\projects\phenomic\node_modules\tapable\lib\Tapable.js:69:14)
    at Compiler.<anonymous> (C:\projects\phenomic\node_modules\webpack\lib\CachePlugin.js:40:4)
    at Compiler.applyPluginsAsync (C:\projects\phenomic\node_modules\tapable\lib\Tapable.js:71:13)
    at Compiler.<anonymous> (C:\projects\phenomic\node_modules\webpack\lib\Compiler.js:400:9)
    at DependenciesBlock.onModuleBuildFailed (C:\projects\phenomic\node_modules\webpack-core\lib\NormalModuleMixin.js:315:19)
    at DependenciesBlock.<anonymous> (C:\projects\phenomic\node_modules\webpack-core\lib\NormalModuleMixin.js:228:39)
    at context.callback (C:\projects\phenomic\node_modules\webpack-core\lib\NormalModuleMixin.js:148:14)
    at Object.<anonymous> (C:\projects\phenomic\test-phenomic-theme-base\node_modules\extract-text-webpack-plugin\loader.js:112:13)
    at Compiler.<anonymous> (C:\projects\phenomic\node_modules\webpack\lib\Compiler.js:214:10)
    at C:\projects\phenomic\node_modules\webpack\lib\Compiler.js:403:12
    at Compiler.next (C:\projects\phenomic\node_modules\tapable\lib\Tapable.js:67:11)
    at Compiler.<anonymous> (C:\projects\phenomic\test-phenomic-theme-base\node_modules\extract-text-webpack-plugin\loader.js:78:5)
    at Compiler.next (C:\projects\phenomic\node_modules\tapable\lib\Tapable.js:69:14)
    at Compiler.<anonymous> (C:\projects\phenomic\node_modules\webpack\lib\CachePlugin.js:40:4)
    at Compiler.applyPluginsAsync (C:\projects\phenomic\node_modules\tapable\lib\Tapable.js:71:13)
    at Compiler.<anonymous> (C:\projects\phenomic\node_modules\webpack\lib\Compiler.js:400:9)
    at Compilation.<anonymous> (C:\projects\phenomic\node_modules\webpack\lib\Compilation.js:577:13)
    at Compilation.applyPluginsAsync (C:\projects\phenomic\node_modules\tapable\lib\Tapable.js:60:69)
    at Compilation.<anonymous> (C:\projects\phenomic\node_modules\webpack\lib\Compilation.js:572:10)
    at Compilation.next (C:\projects\phenomic\node_modules\tapable\lib\Tapable.js:67:11) +3.7s
```

This error happened after Webpack 2 & 1 support has been introduced. Seems to come from extract-text-plugin.

--- a/package.json
+++ b/package.json
@@ -181,7 +181,6 @@
     "tests": "ava",
     "#integration-tests": "needs this order (index test the build, and cli is cleaning it)",
     "integration-tests": "ava __tests__/index.js && ava __tests__/cli.js",
-    "pretest-phenomic-theme-base": "npm run transpile",
     "test-phenomic-theme-base": "babel-node scripts/test-phenomic-theme-base.js && cd test-phenomic-theme-base && npm test",
     "posttest-phenomic-theme-base": "npm run integration-tests",
     "docs": "babel-node scripts/docs.js",

--- a/src/builder/webpack/config.node.js
+++ b/src/builder/webpack/config.node.js
@@ -21,8 +21,10 @@ const defaultExternals = [
 ]
 
 const sourceMapSupport = require.resolve("source-map-support/register")
-// require split to avoid babel-register trying to parse that...
-const requireSourceMapSupport = "requi" + "re('" + sourceMapSupport + "');"
+  // windows support
+  .replace(/\\/g, "/")
+
+const requireSourceMapSupport = "require('" + sourceMapSupport + "');"
 
 export default (config: PhenomicConfig): WebpackConfig => {
   const webpackConfig = commonWebpackConfig(config)

